test/cleanup-test-namespace.sh

```diff
@@ -11,10 +11,10 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 # Delete Aeropsike clusters
 echo "Removing Aerospike clusters"
-kubectl -n test delete aerospikecluster --all
-kubectl -n test1 delete aerospikecluster --all
-kubectl -n test2 delete aerospikecluster --all
-kubectl -n test3 delete aerospikecluster --all
+kubectl -n test delete aerospike-cluster --all
```

Comment:
These should be aerospikecluster. It is to delete all objects of CR type aerospikecluster.